### PR TITLE
add tests for the account balances page

### PR DESCRIPTION
**Pull request by eshaben**
This adds tests for the snippets on the [Account Balances](https://docs.moonbeam.network/builders/get-started/eth-compare/balances/) page of the docs

### CODE REVIEW

**Comment by wuzhong-papermoon** [comment]
I am having issue with "Polkadot.js API test"

```
  const getApi = async () => {
    // Construct API provider
    const wsProvider = new WsProvider('ws://localhost:9944');
    const api = await ApiPromise.create({ provider: wsProvider });
    console.log(api)

    return api;
  };
```
It seems `ApiPromise.create({ provider: wsProvider })` is not working. I am looking into this. The weirder thing is when this fails, it reruns from the previous line `const wsProvider = new WsProvider('ws://localhost:9944');` And hence start a loop of error.

I am looking into the issue.

File: test/builders/get-started/eth-compare/account-balances.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+describe('Account Balances', () => {
+  const alice = {
+    address: '0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac',
+    pk: '0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133',
+  };
+
+  const getApi = async () => {
+    // Construct API provider
+    const wsProvider = new WsProvider('ws://localhost:9944');
```
Comment (wuzhong-papermoon) [inline test/builders/get-started/eth-compare/account-balances.js:14]: In my specific case, `ws://localhost:9944` returns connection refuse error.
I need to use `ws://127.0.0.1:9944` Not sure if it is a general issue though.
How about using WS_RPC_ENDPOINT in .env file?

```suggestion
    const wsProvider = new WsProvider('ws://127.0.0.1:9944');
```

Comment (eshaben) [reply test/builders/get-started/eth-compare/account-balances.js:14]: I can change it for now. This change has been made in a separate PR. Unfortunately, this change relies on Opslayer updating some tests and I've opened a ticket for it, but the test suite isn't super high priority so it hasn't been fixed yet. You can see the change in this PR: https://github.com/papermoonio/moonbeam-docs-test-suite/pull/16/files#diff-a714eca19b6f8db4dbfbd978aa5325b98cc0550fe4c1e7953e8b1eb136cb925aR3

**Review by wuzhong-papermoon — APPROVED** [review]
Great job!

**Pull request merged**